cmd/main.go

```diff
@@ -276,10 +276,13 @@ func main() {
 			cmd = append(cmd[:2], cmd[1:]...)
 			cmd[1] = "--internal-userns"
 
-			forward := make(chan os.Signal)
+			forward := make(chan os.Signal, 3)
 			signal.Notify(forward, syscall.SIGTERM, syscall.SIGHUP, syscall.SIGINT)
 
 			c, err := container.MaybeRunInUserns(cmd)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Error in MaybeRunInUserns(cmd: %s): %s", cmd, err)
```

Comment:
Can we return this error here instead of just printing it so we get a full stack trace?

Reply:
I just went ahead and force pushed this fix since I'd also like to drop the hard-coded version as well. Thanks for these!